homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/style.css

```diff
@@ -26,3 +26,62 @@
 .about-me__links {
   color: #EF4934;
 }
+
+.about-me__article {
+  display: grid;
+  grid-template-columns: auto auto;
+  width: 1175px;
+  height: 479px;
+  left: 53px;
+  border-radius: 22px;
+}
+
+.about-me__picture {
+  width: 532px;
+  height: 356px;
+  left: 53px;
+  margin-top: 48px;
+  margin-left: 53px;
+  border-radius: 22px;
+}
+
+.about-me__header {
+  display: grid;
+  grid-template-columns: 10fr 1fr 1fr 1fr;
+  align-content: center;
+  background-color: #EF4934;
+  width: 1280px;
+  height: 80px;
+  font-family: Roboto;
+  text-decoration: none;
+}
+
+.header__link {
+  text-decoration: none;
+  color: #FFF;
+}
+
+.header__link:hover {
+  color: #1C2321;
+  text-decoration: underline;
+}
+
+.about-me__logo {
+  width: 55px;
+  height: 57px;
```

Comment:
This image is blurred a bit. Because of just on 1 pixel. You can check its initial size simply hovering the image 
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/f7fe66a5-94dd-4a59-9d03-75318607a27d)

Reply:
could U please clafify this point? should I replase logo with other picture format e.g. .img?

Reply:
Yep, you've got it right `height: 57 -> 58`
Now it is ok